docs/topics/OAuth2.md

```diff
@@ -253,7 +253,7 @@ Bot authorization is a special server-less and callback-less OAuth2 flow that ma
 ###### URL Example
 
 ```
-https://discord.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=1
+https://discord.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0
```

Comment:
Why does this need to be changed? It shows an example for giving the bot the create invite permission when inviting.

Reply:
i was just changing the perms